config/routes.rb

```diff
@@ -42,6 +42,8 @@
   delete "/cart", to: "cart#empty"
   delete "/cart/:item_id", to: "cart#remove_item"
   patch "/cart/:item_id", to: "cart#update"
+  patch "/cart/:item_id/decrease", to: "cart#decrease"
+
```

Comment:
When I made the route on 44 and the update action in the controller I thought there might be a way to use the same action for decreasing, which is why I called the action just update. Im not sure about this, I think that its ok to approve for right now and we can ask about it in the next checkin